automation/pxf_regress/main.go

```
@@ -391,37 +337,33 @@ func listTestQueries(testDir string) ([]string, error) {
 	// Sort the tests in ascending order
 	sort.Strings(tests)
 
-	return tests, nil
+	return tests
 }
 
-func printSummary(w *bufio.Writer, successCount int, failCount int) {
-	var summary string
+func getSummary(successCount int, failCount int) string {
 	if failCount == 0 {
-		summary = fmt.Sprintf(" All %d tests passed. ", successCount)
-	} else {
-		summary = fmt.Sprintf(" %d of %d tests failed. ", failCount, successCount+failCount)
+		return fmt.Sprintf("all %d tests passed. ", successCount)
 	}
-	n := len(summary)
 
-	fmt.Fprintln(w)
-	fmt.Fprintln(w, strings.Repeat("=", n))
-	fmt.Fprintln(w, summary)
-	fmt.Fprintln(w, strings.Repeat("=", n))
-	fmt.Fprintln(w)
-	fmt.Fprintln(w)
-
-	w.Flush()
+	return fmt.Sprintf("%d of %d tests failed. ", failCount, successCount+failCount)
 }
 
+const nanosecondsPerMicrosecond = 1000
+
 // Helper function to format timestamp including microseconds without '.'
 //
-// Go's time.Format cannot create strings with milliseconds without '.' between
+// Go's time.Format cannot create strings with millisecond precision without
+// '.' between
 // the seconds and milliseconds so we have to manually remove it.
 func formatTimestamp(t time.Time) string {
-	microsecond := t.Nanosecond() / 1000
+	microsecond := t.Nanosecond() / nanosecondsPerMicrosecond
 	formatted := fmt.Sprintf("%04d%02d%02d%02d%02d%02d%06d",
 		t.Year(), t.Month(), t.Day(),
 		t.Hour(), t.Minute(), t.Second(), microsecond)
 
 	return formatted
 }
+
+func deleteResultsFiles() {
+	os.Remove(regressionDiffsName)
```

I guess we're ignoring the error if the file to be deleted does not exist ?